app/assets/javascripts/components/dropdown.js

```diff
@@ -0,0 +1,44 @@
+function Dropdown ($module) {
+  if (!$module) $module = document
+  this.$module = $module
+  this.$title = $module.querySelector('.app-c-dropdown__title')
+  this.$container = $module.querySelector('.app-c-dropdown__container')
+  this.$buttons = $module.querySelectorAll('.app-c-dropdown__button')
+}
+
+Dropdown.prototype.handleBlur = function (event) {
+  var target = event.relatedTarget
+  if (!target) {
+    target = document.activeElement
+  }
+
+  if (!this.$module.contains(target)) {
+    this.closeDropdown()
+  }
+}
+
+Dropdown.prototype.handleButtonClick = function (event) {
+  this.closeDropdown()
+}
+
+Dropdown.prototype.closeDropdown = function (event) {
+  if (this.$module.hasAttribute('open')) {
```

Comment:
I'm guessing we need this for when we tab over the dropdown?

Reply:
Yep